README.md

```diff
@@ -127,6 +127,7 @@ For the binaries to operate properly on your system, the following conditions ha
 	-	Change the permissions on the sample application “HevcEncoderApp” executable by running the command: 				chmod +x HevcEncoderApp
 	-	cd into your chosen location
 	-	Run the sample application to encode.
+	-	Sample application supports reading from pipe. E.g. ffmpeg -i [input.mp4] -nostdin -f rawvideo -pix_fmt yuv420p - | ./HevcEncoderApp -i stdin -n [number_of_frames_to_encode] -w [width] -h [height].
 
 # Demo features and limitations
 
```

Comment:
Add "For example two channels encoding on Windows: ebHevcEncApp.exe -nch 2 -c firstchannel.cfg secondchannel.cfg" after Multi-instance support section.